document/CoreFormat.mdk

```diff
@@ -1107,8 +1107,10 @@ A `fileChange` object represents a change to a single file.
 
 ### `uri` property { #fileChange-uri }
 
-A `fileChange` object shall contain a property named `uri` whose value is a string value that represents
-the location of the file as a valid URI.
+A `fileChange` object may contain a property named `uri` whose value is a string value that represents
+the location of the file as a valid URI. If the 'uri' property is not present, its value is assumed
+to be the uri that is associated with the current result (i.e., the uri present on the result 
```

Comment:
But on a substantive note: A `result` can have multiple `locations`. We could specify that the `uri` property defaults to the `resultFile`-or-`analysisTarget` of `locations[0]`. But I worry that this spec is complicated for implementers, and I'm not sure how much value it has. Thoughts?